server/utils/index.js

```diff
@@ -164,3 +165,18 @@ export const validateDeck = (deckCards) => {
 
 	if (deckCards.length !== 42) return 'Deck must have exactly 42 cards.'
 }
+
+/**
+ *
+ * @param {Array<Player>} players
+ * @param {string} type
+ * @param {*} payload
+ */
+export function broadcast(players, type, payload = {}) {
```

Comment:
I'd put this into its own utils file, something like `utils/socket` or `utils/messaging` or something like that.